### PR TITLE
[MRG] Add doc note to annotations_from_events function

### PR DESCRIPTION
**Pull request by adam2392**
#### Reference issue
Closes: #8457 

#### What does this implement/fix?
Adds a section on notes for `annotations_from_events` to remind users to invert dictionary.

### CODE REVIEW

File: mne/annotations.py [hunk 1]
```diff
@@ -1098,6 +1098,16 @@ def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
     Notes
     -----
     Annotations returned by this function will all have zero (null) duration.
+
+    Creating events from annotations via the function
+    :func:`mne.events_from_annotations` takes in event mappings with
+    key/value pairs as description/ID, whereas ``annotations_from_events``
+    takes in event mappings with key/value pairs as ID/description.
+    If you need to use these together, you can invert the mapping by doing:
+    ..
```
Comment (drammock) [inline mne/annotations.py:1107]:
```suggestion
    If you need to use these together, you can invert the mapping by doing::
```

File: mne/annotations.py [hunk 2]
```diff
@@ -1098,6 +1098,16 @@ def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
     Notes
     -----
     Annotations returned by this function will all have zero (null) duration.
+
+    Creating events from annotations via the function
+    :func:`mne.events_from_annotations` takes in event mappings with
```
Comment (drammock) [inline mne/annotations.py:1103]:
```suggestion
    `mne.events_from_annotations` takes in event mappings with
```
nowadays sphinx can find things in the mne namespace without explicit `:func:`, `:class:` role declarations

File: mne/annotations.py [hunk 3]
```diff
@@ -1098,6 +1098,16 @@ def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
...
+    key/value pairs as description/ID, whereas ``annotations_from_events``
```
Comment (drammock) [inline mne/annotations.py:1104]:
```suggestion
    key→value pairs as description→ID, whereas `mne.annotations_from_events`
```

File: mne/annotations.py [hunk 4]
```diff
@@ -1098,6 +1098,16 @@ def annotations_from_events(events, sfreq, event_desc=None, first_samp=0,
...
+    takes in event mappings with key/value pairs as ID/description.
```
Comment (drammock) [inline mne/annotations.py:1105]:
```suggestion
    takes in event mappings with key→value pairs as ID→description.
```

**Review by agramfort — APPROVED** [review]

**Comment by larsoner** [comment]
Thanks @adam2392

**Pull request merged**